#!/usr/bin/env python
PACKAGE = "twitcher_connection"

gen = ParameterGenerator()

gen.add("oauth_consumer_key", str_t, 0, "The OAuth Consumer Key provided by the Twitter API", "")
gen.add("oauth_consumer_secret", str_t, 0, "The OAuth Consumer Secret provided by the Twitter API", "")
gen.add("oauth_token", str_t, 0, "The OAuth token used for this application", "")
gen.add("oauth_token_secret", str_t, 0, "The corresponding OAuth token for the secret above", "")
gen.add("oauth_version", str_t, 0, "Verion of OAuth used for the API calls", "")